suno/suno.py

```diff
@@ -121,7 +121,7 @@ def _fetch_songs_metadata(self, ids):
         self.now_data = data
         try:
             for d in data:
-                # only get one url for now
+                # only get one url for now TODO: See if possible for both urls
```

Comment:
no need add this TODO cause download is very slow....we decided only download one